pkg/cli/connect.go

```diff
@@ -38,7 +38,8 @@ var connectCmd = &cobra.Command{
 	Short: "Create certificates for securely connecting with clusters\n",
 	Long: `
 Bootstrap security certificates for connecting to new or existing clusters.`,
-	RunE: UsageAndErr,
+	RunE:       UsageAndErr,
+	Deprecated: "self generated certificates are no longer support functionality use '--insecure' instead",
```

Comment:
```suggestion
	Deprecated: "self-generated certificates are no longer supported, use '--insecure' instead",
```

Do we really want to encourage people to use `--insecure`?